locales/templates/app.pot

```diff
@@ -2209,3 +2203,123 @@ msgctxt "msrp_activitykilled_snackbar_body"
 msgid "You’ve won a shopping voucher!"
 msgstr ""
 
+#. Travel Vertical
+#. Artboard Name: 2.1 See bucket list
+#. travel home
+msgctxt "travel_vertical_title"
+msgid "Travel"
+msgstr ""
+
+msgctxt "travel_vertical_category_1"
+msgid "Explore"
+msgstr ""
+
+msgctxt "travel_vertical_category_2"
+msgid "Bucket List"
+msgstr ""
+
+#. featured destinations
+msgctxt "travel_destination_subcategory_1"
+msgid "Travel on a budget"
+msgstr ""
+
+msgctxt "travel_destination_subcategory_2"
+msgid "Best cities for winter holiday"
+msgstr ""
+
+#. Finding destinations
+msgctxt "travel_find_destination_title"
+msgid "Where do you want to go?"
+msgstr ""
+
+msgctxt "travel_find_destination_placeholder_1"
+msgid "Try “Bali”…"
+msgstr ""
+
+#. Subcategories in detail page
+#. The %1$s here is a placeholder for destination name like Bali
+#, c-format
+msgctxt "travel_detail_page_subcategory_content"
+msgid "Explore %1$s"
+msgstr ""
+
+msgctxt "travel_detail_page_subcategory_hotel"
+msgid "Popular hotels"
+msgstr ""
+
+#. IG content. the %1$s here is the destination name
+#, c-format
+msgctxt "travel_content_ig_link"
+msgid "#%1$s on IG"
+msgstr ""
+
+#. YT content. the %1$s here is how many times of video viewing
+#, c-format
+msgctxt "travel_content_yt_view_count"
+msgid "%1$s views"
+msgstr ""
+
+#. Wiki content
+msgctxt "travel_content_wiki_source_name"
+msgid "Wikipedia"
+msgstr ""
+
+#. detail information for the hotel room
+msgctxt "travel_hotel_detail_1"
+msgid "Free Wi-Fi"
+msgstr ""
+
+msgctxt "travel_hotel_detail_2"
+msgid "Free cancellation"
+msgstr ""
+
+msgctxt "travel_hotel_detail_3"
+msgid "Pay upon check in"
+msgstr ""
+
+#. text in the empty state
+msgctxt "travel_empty_state_text"
+msgid "“I haven’t been everywhere, but it’s on my list.”"
+msgstr ""
+
+#. CTA button for the empty state
+msgctxt "travel_empty_state_button"
+msgid "Explore a city"
+msgstr ""
+
+#. Artboard Name: 2.2 Save to a bucket list
+#. Snackbar message after saving a destination
```

Comment:
same comment as above: the first  line doesn't make sense from a l10n comment perspective